### DISCUSSION TITLE
Is there a way to change a package id?

### DISCUSSION THREAD

**Discussion by marticliment** [microsoft/winget-pkgs#82911] [Q&A] [answered]
Is there any way to change one package ID? 
Like i'd like to change `OldAuthor.PackageName` to `NewAuthor.PackageName`.

Thanks!

**Comment by mdanish-kh** [comment, marked as answer]
You can learn about the process [here](https://github.com/microsoft/winget-pkgs/blob/master/FAQ.md#what-should-i-do-if-a-package-is-being-published-by-a-new-publisher) and see related PRs https://github.com/microsoft/winget-pkgs/pull/66947 and https://github.com/microsoft/winget-pkgs/pull/66948 to learn how it was carried out.
